puppet/zulip/files/nagios_plugins/zulip_nagios_server/check_postgres_replication_lag

```diff
@@ -24,12 +24,13 @@ def report(state, msg):
 def get_loc_over_ssh(host, func):
     # type: (str, str) -> str
     try:
-        return subprocess.check_output(['ssh', host,
-                                        'psql zulip -t -c "SELECT %s()"' % (func,)],
-                                       stderr=subprocess.STDOUT,
-                                       universal_newlines=True)
+        output = subprocess.check_output(['ssh', host,
+                                          'psql zulip -t -c "SELECT %s()"' % (func,)],
+                                         stderr=subprocess.STDOUT,
+                                         universal_newlines=True)
     except subprocess.CalledProcessError as e:
         report('CRITICAL', 'ssh failed: %s: %s' % (str(e), e.output))
+    return output
```

Comment:
This is probably the wrong fix.  What's going on here is that `report`, unknown to mypy, will unconditionally exit.  I think we may want to `type: ignore` this for now and we can ask the mypy folks about it -- maybe there's a tag we're supposed to put on `report` to show that it exits unconditionally.